apps/agent/test/agent_shared.js

```diff
@@ -172,6 +194,72 @@ module.exports = (
         await assertRevert(agent.initialize(), errors.INIT_ALREADY_INITIALIZED)
       })
 
+      context('> ERC165', () => {
+        it('supports base ERC165 interface', async () => {
+          assert.isTrue(await agent.supportsInterface(ERC165_SUPPORT_INTERFACE_ID))
+          assert.isFalse(await agent.supportsInterface(ERC165_SUPPORT_INVALID_ID))
+        })
+
+        it('supports ERC1271 interface', async () => {
+          assert.isTrue(await agent.supportsInterface(ERC1271_INTERFACE_ID))
+        })
+
+        it('supports ERC721Receiver interface', async () => {
+          assert.isTrue(await agent.supportsInterface(ERC721_RECEIVED_INTERFACE_ID))
+        })
+
+        it('supports ERC1155Receiver interface', async () => {
+          assert.isTrue(await agent.supportsInterface(ERC1155_RECEIVER_INTERFACE_ID))
+        })
+
+        it("doesn't support any other interface", async () => {
+          assert.isFalse(await agent.supportsInterface('0x12345678'))
+          assert.isFalse(await agent.supportsInterface('0x'))
```

Comment:
Requested by EIP 165
```suggestion
          assert.isFalse(await agent.supportsInterface('0x'))
          assert.isFalse(await agent.supportsInterface('0xffffffff'))
```